src/client/screens/CreateProjectScreen.tsx

```diff
@@ -32,12 +32,16 @@ interface StateProps {
 }
 
 const validate = (form: ProjectForm) =>
-  form.name.trim() !== "" && form.numberOfDistricts !== null && form.regionConfig !== null
+  form.name.trim() !== "" &&
+  form.numberOfDistricts !== null &&
+  form.regionConfig !== null &&
+  form.chamber
```

Comment:
It is acceptable for `chamber` to be null (custom districts)
```suggestion
  form.regionConfig !== null
```